t/33-vagrant.t

```diff
@@ -190,8 +190,7 @@ subtest 'backend creation dies when VAGRANT_ASSETDIR cannot be opened' => sub {
     $backend_vars->{VAGRANT_ASSETDIR} = $fakedir;
     $backend_vars->{VAGRANT_BOX} = "/foobar";
 
-    eval { backend::vagrant->new(); };
-    like($@, qr/Could not opendir $fakedir/, 'opendir error message is present in the output');
+    throws_ok { backend::vagrant->new() } qr/Could not opendir $fakedir/, 'opendir error message is present in the output';
```

Comment:
`Could not opendir` seems like it meant to be `Could not open dir`?! which if this is the case i would like to request a better message

Reply:
I honestly didn't even read the string. If you care then please investigate what is meant here in detail and propose a separate PR

Reply:
`opendir` is a perl function